cli/dcoscli/job/main.py

```diff
@@ -228,6 +233,39 @@ def _remove(job_id, stop_current_job_runs=False):
     return 0
 
 
+def _queue(job_id, json_flag=False, quiet=False):
+    """
+    :param job_id: Id of the job
+    :type job_id: str
+    :param json_flag: output json if True
```

Comment:
It should start with a capital `O`, based on the style of other docstrings in this file.